src/main/resources/templates/entries.html

```diff
@@ -22,7 +22,7 @@ <h1 class="heading-large">
             <span class="heading-secondary"
                   th:include="fragments/pagination.html::pagination-top('entries')"></span>
           </h1>
-          <div th:include="fragments/download-and-preview.html::download-and-preview('entries')"></div>
+          <div th:include="fragments/download-and-preview.html::download-and-preview(${'record/' + content.recordKey.get() + '/entries'})"></div>
```

Comment:
I believe this should be `/entries` and if this is being used to show the /entries page. This is currently returning a 500 and failing tests @karlbaker02

Reply:
This makes sense. I changed this earlier on because downloads of entries for individual keys downloaded _all_ entries, not just what is being viewed (FYI this is also the case in production, so an existing bug).

Reply:
I will revert this for now and we can fix later on.